ops/scripts/integration-tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+RETRIES=${RETRIES:-60}
+JSON='{"jsonrpc":"2.0","id":0,"method":"rollup_getInfo","params":[]}'
+
+if [[ ! -z "$URL" ]]; then
+    # get the addrs from the URL provided
+    ADDRESSES=$(curl --silent --retry-connrefused --retry $RETRIES --retry-delay 5 $URL)
+    export ADDRESS_MANAGER=$(echo $ADDRESSES | jq -r '.AddressManager')
+fi
+
+# wait for the sequencer to be up
+curl \
+    --silent \
+    -H "Content-Type: application/json" \
+    --retry-connrefused \
```

Comment:
Could we add a `--output /dev/null` here because the result of this will show up in the logs otherwise